### PR TITLE
FUSETOOLS-3516: Convert fixed UI Test 'ConfigurationsEditorDataFormat…

### PR DESCRIPTION
**Pull request by pospisilf**
…Test' into master branch

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [ ] Did you use the Jira Issue number in the commit comments?
- [ ] Did you set meaningful commit comments on each commit?
- [ ] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

**Comment by pospisilf** [comment]
rebased

**Review by apupier — APPROVED** [review]

**Review by djelinek — APPROVED** [review]

**Comment by apupier** [comment]
to have a succesful build, need to be rebased on main branch

**Comment by pospisilf** [comment]
rebased

File: uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/ConfigurationsEditorDataFormatTest.java [hunk 1]
```diff
@@ -306,14 +308,23 @@ private void assertDataFormatPath(boolean expected) {
...
+	public boolean checkForReportedParameters() {
+		if (new JiraClient().isIssueClosed("FUSETOOLS-1930")) {
+			return false;
+		} else {
+			if (dataFormat.equals(
+					"zipfile - The Zip File data format is a message compression and de-compression format of zip files.")) {
+				return true;
+			} else {
+				return false;
+			}
```
Comment (apupier) [inline uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/ConfigurationsEditorDataFormatTest.java:327]:
```suggestion
			return "zipfile - The Zip File data format is a message compression and de-compression format of zip files.".equals(dataformat);
```

it avoids potential NPE if dataformat is null.
less line of code.

**Review by apupier — APPROVED** [review]

**Review by djelinek — APPROVED** [review]

File: uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/ConfigurationsEditorDataFormatTest.java (hunk 1 above)
Comment (pospisilf) [reply uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/ConfigurationsEditorDataFormatTest.java]: Good point, changed. Thanks!

**Comment by pospisilf** [comment]
testPR

**Review by djelinek — DISMISSED** [review]

**Comment by djelinek** [comment]
testPR

**Review by djelinek — APPROVED** [review]

**Pull request merged**